lib/game_stats.rb

```diff
@@ -0,0 +1,35 @@
+# require "./lib/stattracker.rb"
+require "csv"
+
+lines = File.readlines'./data/games_fixture.csv'
```

Comment:
Cool so I think we'll pull this into StatTracker's ::from_csv stuff later on until we figure out if/how we want to refactor that all.